src/gui/components/TextInput.zig

```diff
@@ -514,7 +519,17 @@ pub fn render(self: *TextInput, mousePosition: Vec2f) void {
 			draw.setColor(0x440000ff);
 			self.textBuffer.drawSelection(textPos, @min(selectionStart, cursor), @max(selectionStart, cursor));
 		}
-		draw.setColor(0xff000000);
-		draw.line(cursorPos, cursorPos + Vec2f{0, 16});
+
+		const milliTime = main.game.world.?.milliTime;
```

Comment:
This time is inaccurate. It does not show the time of the last frame.
I'd suggest to instead just use std.time directly.